evennia/server/evennia_launcher.py

```diff
@@ -1480,14 +1478,118 @@ def create_superuser():
+        UserAttributeSimilarityValidator,
+        MinimumLengthValidator,
+        CommonPasswordValidator,
+        NumericPasswordValidator,
+    )
+    from django.core.exceptions import ValidationError
+    from django.contrib.auth.validators import ASCIIUsernameValidator
+    from django.core.validators import MinLengthValidator, MaxLengthValidator
+    from evennia.server.validators import EvenniaUsernameAvailabilityValidator
+    
+    username_validators = [
+        ASCIIUsernameValidator(),
+        MinLengthValidator(3),
+        MaxLengthValidator(30),
+        EvenniaUsernameAvailabilityValidator(),
+    ]
+    password_validators = [
+        UserAttributeSimilarityValidator(),
+        MinimumLengthValidator(8),
+        CommonPasswordValidator(),
+        NumericPasswordValidator()
+    ]
+    
+    username_errors = []
+    for validator in username_validators:
+        try:
+            # ASCIIUsernameValidator uses __call__ instead of validate
+            if hasattr(validator, 'validate'):
+                validator.validate(account_name, user=None)
+            else:
+                # For validators that use __call__ like ASCIIUsernameValidator
+                validator(account_name)
+        except ValidationError as e:
+            username_errors.extend(e.messages)
+    if username_errors:
+        print("Username validation error:")
+        for error in username_errors:
+            print(f"  - {error}")
+        return False
+    
+    print(f"\nCreating account '{account_name}'")
+    email = input("Email (optional): ")
+    
+    while True:
+        password = input("Password: ")
+        if not password:
+            print("Password cannot be empty.")
+            continue
+            
+        password_errors = []
+        for validator in password_validators:
+            try:
+                validator.validate(password, user=None)
+            except ValidationError as e:
+                password_errors.extend(e.messages)
+        
+        if password_errors:
+            print("Password validation error:")
+            for error in password_errors:
+                print(f"  - {error}")
+            continue
+            
+        password2 = input("Password (again): ")
+        if password != password2:
+            print("Passwords don't match.")
+            continue
+            
+        break
+    
+    account = AccountDB.objects.create_account(key=account_name, email=email, password=password)
+    account.save()
+    print(f"Account '{account_name}' was created successfully.")
+    return True
+
+def run_create_user(account_name):
+    """
+    Run the create_user function from the command line.
+    
+    Args:
+        account_name (str): Name of the account to create.
+    """
+    from django.core.exceptions import ValidationError
+    
+    try:
+        create_user(account_name)
+    except ValidationError as e:
+        print(f"Error creating account: {e}")
```

Comment:
Is the only reason for splitting this into two functions, the ability to wrap the innermost function in `try...except` ?